udata/tags.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from slugify import slugify
+
+
+def slug(value):
```

Comment:
I prefer `normalize` because slug means "url part for permalink/pretty URLs".
I think at some we might have some different rules for normalized tags and slugs